CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Grafana Mimir
 
 * [FEATURE] Ingester/Distributor: Add support for exporting cost attribution metrics (`cortex_ingester_attributed_active_series`, `cortex_distributor_received_attributed_samples_total`, and `cortex_discarded_attributed_samples_total`) with labels specified by customers to a custom Prometheus registry. This feature enables more flexible billing data tracking. #10269
+* [CHANGE] Ingester/Distributor: Prefix cost attribution metric labels with `attributed_` to prevent collisions with system cluster and namespace labels. #10509
```

Comment:
This is not needed because this new feature isn't included in any release yet.

Reply:
addressed here 97a5d77efb0c7b63dd34d6fa8b5d2216d7334373